.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -5,66 +5,102 @@ labels: [bug]
 body:
   - type: markdown
     attributes:
-      value: |
+      value: >
         ### Thanks for taking the time to fill out this bug report!
-        **Before filling out this report**, please check our [Troubleshooting](https://pptr.dev/troubleshooting) guide for solutions to common issues.
-        Also, verify the problem is *related to packages maintained by Puppeteer* (this does not include `puppeteer-extra`, `expect-puppeteer`, etc.).
-  - type: textarea
-    id: summary
-    attributes:
-      label: Bug description
-      description: What did you do? What did you expect to happen? What actually happened instead? Provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
-      value: |
-        Steps to reproduce the problem:
 
-        1. …
+        **Before filling out this report**, please check our
+        [Troubleshooting](https://pptr.dev/troubleshooting) guide for solutions
+        to common issues.
+  - id: summary
+    type: textarea
+    attributes:
+      label: Bug expectation
```

Comment:
What about splitting this in two -> Expected result & Actual result?

Reply:
We only care about expectation. Their code should show the actual results since we will run and gather artifacts.